### ISSUE TITLE
not mangling not working

### ISSUE THREAD

**Issue by incanus** [CocoaPods/cocoapods-packager#110]
What am I doing wrong here? 

``` bash
$ pod package --no-mangle Mapbox-iOS-SDK
[!] podspec has binary-only depedencies, mangling not possible.
```

I've tried `--library` and `--embedded` as well, but no change.

**Comment by segiddins** [comment]
This is a known issue, will be fixed by https://github.com/CocoaPods/cocoapods-packager/pull/107.

**Comment by incanus** [comment]
:+1:

**Comment by manuyavuz** [comment]
Fixed by #107.
